plat/layerscape/board/ls1043/platform.mk

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+# indicate the reset vector address can be programmed
+PLAT_XLAT_TABLES_DYNAMIC	:=	1
```

Comment:
Actually, I can't see the dynamic mapping functions being used at all. You should set this to 0. I can see that in `layerscape/common/ls_common.mk` you also have a conditional to use the xlat tables lib v1 or v2, you should remove the compatibility with v1. We only have the build option `ARM_XLAT_TABLES_LIB_V1` so that we can do internal testing of both versions of the library. Unless you have a good reason to use v1, if you just want to make your platform future-proof, stick to v2.

Reply:
Removed PLAT_XLAT_TABLES_DYNAMIC and uses xlat library v2, thanks.